controllers/datadogagent/feature/tcp_queue_length/feature.go

```diff
@@ -0,0 +1,108 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package tcpqueuelength
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/DataDog/datadog-operator/apis/datadoghq/v1alpha1"
+	"github.com/DataDog/datadog-operator/apis/datadoghq/v2alpha1"
+	apiutils "github.com/DataDog/datadog-operator/apis/utils"
+
+	apicommon "github.com/DataDog/datadog-operator/apis/datadoghq/common"
+	apicommonv1 "github.com/DataDog/datadog-operator/apis/datadoghq/common/v1"
+	"github.com/DataDog/datadog-operator/controllers/datadogagent/feature"
+	"github.com/DataDog/datadog-operator/controllers/datadogagent/object/volume"
+)
+
+func init() {
+	err := feature.Register(feature.TCPQueueLengthIDType, buildTCPQueueLengthFeature)
+	if err != nil {
+		panic(err)
+	}
+}
+
+func buildTCPQueueLengthFeature(options *feature.Options) feature.Feature {
+	tcpQueueLengthFeat := &tcpQueueLengthFeature{}
+
+	return tcpQueueLengthFeat
+}
+
+type tcpQueueLengthFeature struct {
+	enable bool
+	owner  metav1.Object
```

Comment:
it seems that `owner` is not needed. can we remove it?

Reply:
🤦‍♀️